### DISCUSSION TITLE
0.16-alpha

### DISCUSSION THREAD

**Discussion by jasonjgardner** [jasonjgardner/jg-rtx#25] [Screenshots]
A couple screenshots from the next release:

> ![JG RTX terracotta](https://user-images.githubusercontent.com/1903667/113967325-5fcdbd80-97f6-11eb-82b8-f01e59569048.png)
> Glazed terracotta design

> ![JG RTX grass](https://user-images.githubusercontent.com/1903667/113967447-9e637800-97f6-11eb-9f12-f1439e3e25e0.png)
> Longer grass

**Comment by erich666** [comment]
Groovy terracotta - looking forward to trying it out. Thanks for this. I'm hoping for a new version of Mineways, with this new content, sometime next week.
